### DISCUSSION TITLE
RenderDoc/CPP/markdown

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#672] [Announcements]
# RenderDoc/CPP/markdown

VuePress 主要从 Markdown 文件生成页面。因此，你可以使用它轻松生成文档或博客站点。 你应该创建和编写 Markdown 文件，以便 VuePress 可以根据文件结构将它们转换为不同的页面。 Markdown 介绍 如果你是一个新手，还不会编写 Markdown，请先阅读 Markdown 介绍 (https://theme-hope...

https://rendertool.github.io/RenderDoc/CPP/markdown.html

**Comment by RenderTool** [comment]
6666666

**Reply by Mister-Hope** [reply]
https://github.com/vuepress-theme-hope/giscus-discussions?tab=readme-ov-file#readme
